src/main/java/com/faforever/api/data/domain/Login.java

```diff
@@ -11,7 +13,11 @@
 import javax.persistence.Inheritance;
 import javax.persistence.InheritanceType;
 import javax.persistence.MappedSuperclass;
-import javax.persistence.OneToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Transient;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
 
 @MappedSuperclass
 @Inheritance(strategy = InheritanceType.JOINED)
```

Comment:
I did not touch this code .... but maybe you are right ... http://stackoverflow.com/a/9669719
`@MAppedSuperclass` should be enough ...